packages/components/src/models/PlayingCard.ts

```diff
@@ -0,0 +1,31 @@
+import type { Card } from '@models/Card';
+
+// eslint-disable-next-line no-shadow
+export enum PlayingCardSuit {
+  Spades = 'Spades',
+  Hearts = 'Hearts',
+  Diamonds = 'Diamonds',
+  Clubs = 'Clubs',
+}
+
+// eslint-disable-next-line no-shadow
+export enum PlayingCardName {
```

Comment:
Actually I checked and I need it to generate code when it is transpiled because of the way I generate a random card or a random card deck, as you can see on the screenshot
![image](https://user-images.githubusercontent.com/10576609/100539441-c2a2bd00-3236-11eb-96de-0fa322c79628.png)